### DISCUSSION TITLE
/posts/technicaltutorials/%E6%96%B0%E4%B8%80%E4%BB%A3%E9%9D%99%E6%80%81%E5%8D%9A%E5%AE%A2%E6%A1%86%E6%9E%B6astro%E7%9A%84%E9%83%A8%E7%BD%B2%E4%BC%98%E5%8C%96%E6%8C%87%E5%8D%97%E4%B8%8E%E4%BD%BF%E7%94%A8%E4%BD%93%E9%AA%8C/

### DISCUSSION THREAD

**Discussion by giscus** [Lapis0x0/blog-discussion#2] [Announcements]
# 
/posts/technicaltutorials/%E6%96%B0%E4%B8%80%E4%BB%A3%E9%9D%99%E6%80%81%E5%8D%9A%E5%AE%A2%E6%A1%86%E6%9E%B6astro%E7%9A%84%E9%83%A8%E7%BD%B2%E4%BC%98%E5%8C%96%E6%8C%87%E5%8D%97%E4%B8%8E%E4%BD%BF%E7%94%A8%E4%BD%93%E9%AA%8C/

本文介绍了新一代静态博客框架Astro的迁移优化步骤与使用体验，阐明了Astro的轻量化特点、灵活性及其独特的群岛架构，结合fuwari主题，提供了详细的自定义和部署指南，使开发者能够轻松构建高性能的博客。

https://www.lapis.cafe/posts/technicaltutorials/%E6%96%B0%E4%B8%80%E4%BB%A3%E9%9D%99%E6%80%81%E5%8D%9A%E5%AE%A2%E6%A1%86%E6%9E%B6astro%E7%9A%84%E9%83%A8%E7%BD%B2%E4%BC%98%E5%8C%96%E6%8C%87%E5%8D%97%E4%B8%8E%E4%BD%BF%E7%94%A8%E4%BD%93%E9%AA%8C/

**Comment by MiaoWuNYA** [comment]
这个框架好漂亮，等有空我也去折腾下（AVA）

**Reply by Lapis0x0** [reply]
好看还性能优秀！

**Comment by anfevs** [comment]
好强好强

**Comment by Keke0522** [comment]
1
